humanguide/other-design-considerations/scaling-images.rst

```diff
@@ -0,0 +1,27 @@
+Scaling images
+==============
+
+Scaling images is important in a convergent app. App icons and images can be created in scalable formats to improve their ability to scale.
+
+.. figure:: /_static/images/humanguide/53.png
+   :align: center
+
+One of the most popular scalable formats is `SVG <https://en.wikipedia.org/wiki/Scalable_Vector_Graphics>`__. SVG files can be created using software like `Inkscape <https://inkscape.org/>`__.
+
+One particularity of qml, though, is that svg are rendered to use low memory and losing the advatadges of scalability. When using a SVG file in an ``Image`` element we can use the property called ``sourceSize``. This will define the rendered size of the SVG image. A SVG image with a ``sourceSize`` width and height of 252 pixels, will be rendered as if the scalable image would be a png of 252 per 252 pixels.
+
+.. figure:: /_static/images/humanguide/icon-sizes.png
+   :align: center
+
+   Different redered sizes of the same svg image
+
+It is not advisable to define ``sourceSize`` as a relative size to avoid performance issues. A good practice could be defining different ``sourceSize`` depending on diverse thresholds. In the example below the image.svg will be rendered 60 per 60 grid units if the main view is bigger that 70 grid units but as an image of 40 per 40 grid units in the other cases. Like this we make use of the image scability and no performance cost.
```

Comment:
"A good practice is defining `sourceSize` based on a set list of thresholds."

...

"By defining these thresholds, we get scalable images without a huge performance cost."